eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -112,7 +112,7 @@ jobs:
         $(CommonMSBuildArgs)
         $(MsbuildSigningArguments)
 
-  - ${{ if eq(parameters.osGroup, 'OSX') }}:
+  - ${{ if or(eq(parameters.osGroup, 'OSX'), eq(parameters.osGroup, 'iOS')) }}:
```

Comment:
You can simplify this condition by using `in` conditional expression.

```yml
${{ if in(parameters.osGroup, 'OSX', 'iOS') }}:
```

That way you don't need to do an or of eq statements.